apps/manifest.go

```diff
@@ -250,6 +262,19 @@ func (m Manifest) IsValid() error {
 		}
 	}
 
+	for _, v := range []validator{
+		m.AppID,
+		m.Version,
+		m.AppType,
+		m.RequestedPermissions,
+	} {
+		if v != nil {
+			if err := v.Validate(); err != nil {
```

Comment:
This returns the first error - maybe in the future we iterate over all values and return the set of errors (to return/print out all places where the manifest is invalid).

Reply:
I agree, I think this should be done in this PR or ticketed

Reply:
Done, replaced with `multierror` in Validate, as appropriate